aks-deploy/README.md

```diff
@@ -0,0 +1,41 @@
+ 
+# GitHub Action for deploying to Azure K8S Service
+
+
+## Usage
+
+```
+
+
+action "Deploy to Azure Kubernetes Service" {
+  uses = "actions/azure/aks@master"
+  secrets = ["DOCKER_PASSWORD", "KUBE_CONFIG_DATA"]
+  env = {
+    AKS_CLUSTER_NAME = "<AKS Cluster Name>"
+    DOCKER_USERNAME = "<Docker Registry username>"
+    DOCKER_REGISTRY_URL = "<Docker Registry URL>"
+    CONTAINER_IMAGE_NAME = "<Container Image Name>" 
+  }
+}
+
+```
+
+
+### Secrets
+
+- `DOCKER_PASSWORD` – **Optional** if public registry or if trust has been established with AKS. This is the password used to log in to your Docker registry. 
+- `KUBE_CONFIG_DATA` - **Optional**. "kubectl config" file content with credentials for Kubernetes to access the cluster. If we use this option, we wont need ["Azure Login"] action as a precursor to this one. 
+
+
+
+### Environment variables
+
+- `AKS_CLUSTER_NAME` – **Required** 
+- `DOCKER_REGISTRY_URL` – **Required** 
+- `DOCKER_USERNAME` – **Optional** if public registry or if trust has been established with AKS. This is the username used to log in to your Docker registry.
+- `CONTAINER_IMAGE_NAME` – **Required** 
+- `CONTAINER_IMAGE_TAG` – **Optional** - ( default value =  "latest").  
+- `HELM_RELEASE_NAME` - **Optional** ( default value =  "aks-deploy").  
+- `HELM_CHART_PATH` - **Optional** ( default value =  ./default-chart) 
+ 
+
```

Comment:
Perhaps you could give examples 
1. With a public registry
2. With a private registry

Otherwise, it's a bit hard to decipher from the input list I suppose.

Reply:
sure, will check this specifically with GH team

Reply:
Resolving this for now. will update if we have feedback from above